opentech/publicpages/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class PublicpagesConfig(AppConfig):
+    name = 'publicpages'
```

Comment:
if I am reading this right, this implies all of the public pages are going into this app?

Reply:
Ping me when the post-restructure rebase is done.